projects/pgai/pgai/vectorizer/loading.py

```diff
@@ -15,13 +16,21 @@ class LoadedDocument:
 
 
 def guess_filetype(file_like: BytesIO, file_path: str | None = None) -> str | None:
-    guess = filetype.guess(file_like)  # type: ignore
+    extension = filetype.guess_extension(file_like)  # type: ignore
```

Comment:
This is a simplification. Calling guess_extension calls guess internally and just returns the extension, which is all we need.